packages/forms-web-app/src/dynamic-forms/controller.js

```diff
@@ -248,16 +248,34 @@ exports.submit = async (req, res) => {
 	);
 };
 
+/**
+ * @type {import('express').Handler}
+ */
+exports.appellantSubmit = async (req, res) => {
```

Comment:
this name feels a bit weird, maybe just call this `submitAppellantSubmission` too?

Reply:
Yep, changed